src/node.cc

```diff
@@ -4010,8 +4010,8 @@ static void ParseArgs(int* argc,
 }
 
 
-static void StartDebug(Environment* env, const char* path,
-                       DebugOptions debug_options) {
+static void StartInspector(Environment* env, const char* path,
```

Comment:
Legacy... when all the debug code was deleted this function was left with its old pre-inspector name.